timon/conf/config.py

```diff
@@ -17,6 +17,7 @@
 import minibelt
 
 from timon import plugins
+from timon.plugins.base import OnDbStorePlugin
```

Comment:
As `isort` of github CI pointed. `timon.plugins.base ` >  `timon.db.store`
so it should be added after the next line.

Reply:
yes, that is why it's a draft

Reply:
also beyond `isort`, it was nice having `timon.plugins`-related import together; but I suppose `isort` wins